helm/microgateway/README.md

```diff
@@ -1,145 +1,164 @@
-
-To access the console in the environment,
-
-a. Obtain the external IP (`EXTERNAL-IP`) of the Ingress resources by listing down the Kubernetes Ingresses.
+## Configuration
 
-```
-kubectl get ing -n <NAMESPACE>
-```
+The following tables lists the configurable parameters of the chart and their default values.
 
-```
-NAME                       HOSTS                ADDRESS        PORTS     AGE
-wso2micro-gw-ingress       <RELEASE_NAME>       <EXTERNAL-IP>  80, 443   3m
-```
+###### WSO2 Subscription Configurations
+| Parameter | Description | Default Value |
+|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------|
+| `wso2.subscription.username` | Your WSO2 Subscription username | "" |
+| `wso2.subscription.password` | Your WSO2 Subscription password | "" |
+  
+If you do not have an active WSO2 subscription, **do not change** the parameters `wso2.subscription.username` and `wso2.subscription.password`.
 
-b. Add the above host as an entry in /etc/hosts file as follows:
 
-```
-<EXTERNAL-IP>	<RELEASE_NAME>
-```
+###### Centralized Logging Configurations 
+| Parameter | Description | Default Value |
+|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------|
+| `wso2.centralizedLogging.enabled` | Enable Centralized logging for WSO2 components | false | | |
+| `wso2.centralizedLogging.logstash.config.logstashYaml` | Logstash custom deployment configuration file | - |
+| `wso2.centralizedLogging.logstash.config.logstashConf` | Logstash custom deployment configuration file | - |
+| `wso2.centralizedLogging.logstash.elasticsearch.username` | Elasticsearch username | elastic |
+| `wso2.centralizedLogging.logstash.elasticsearch.password` | Elasticsearch password | changeme |
+| `wso2.centralizedLogging.logstash.indexNodeID.wso2ISNode` | Elasticsearch IS Node log index ID(index name: ${NODE_ID}-${NODE_IP}) | wso2 |
+  
+###### Micro Gateway Configurations
+| Parameter | Description | Default Value |
+|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|-----------------------------|
+| `wso2.deployment.wso2microgw.dockerRegistry` | Docker registry of the microgateway image | "" |
+| `wso2.deployment.wso2microgw.imageName` | Image name for microgateway node | "" |
+| `wso2.deployment.wso2microgw.imageTag` | Image tag for microgateway node | "" |
+| `wso2.deployment.wso2microgw.replicas` | Number of replicas for microgateway node | 1 |
+| `wso2.deployment.wso2microgw.minReadySeconds` | Refer to [doc](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#deploymentspec-v1-apps)| 1 75 |
```

Comment:
@chathuraswso2 let's get rid of `minReadySeconds` property as it is not required in the Deployment.